backend/test/test_models.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from datetime import datetime
+from geoalchemy2.elements import WKTElement
+from app.models import User, Department, ClearCut
+
+
+def test_user_creation(db):
+    user = User(
+        firstname="Tétras", lastname="Foret", email="tetras.foret@example.com", role="volunteer"
+    )
+    db.add(user)
+    db.commit()
+
+    with pytest.raises(ValueError) as exc_info:
+        User(
+            firstname="Invalid",
+            lastname="User",
+            email="invalid.user@example.com",
+            role="not_a_valid_role",
+        )
+    assert str(exc_info.value) == "Role must be one of: admin, viewer, volunteer"
+
+    assert user.id is not None
+    assert user.created_at is not None
+    assert user.updated_at is not None
+    assert user.deleted_at is None
+
+
+def test_department_creation(db):
+    department = Department(code=75)
+    db.add(department)
+    db.commit()
+
+    assert department.id is not None
+
+
+def test_clear_cut_creation(db):
+    clear_cut = ClearCut(
+        cut_date=datetime.now(),
+        slope_percentage=15.5,
+        location=WKTElement("POINT(48.8566 2.3522)"),
+        boundary=WKTElement(
+            "POLYGON((2.2241 48.8156, 2.4699 48.8156, 2.4699 48.9021, 2.2241 48.9021, 2.2241 48.8156))"
+        ),
+        status="pending",
+    )
+    db.add(clear_cut)
+    db.commit()
+
+    with pytest.raises(ValueError) as exc_info:
+        ClearCut(
+            cut_date=datetime.now(),
+            slope_percentage=15.5,
+            location=WKTElement("POINT(48.8566 2.3522)"),
+            boundary=WKTElement(
+                "POLYGON((2.2241 48.8156, 2.4699 48.8156, 2.4699 48.9021, 2.2241 48.9021, 2.2241 48.8156))"
+            ),
+            status="invalid_status",
+        )
+    assert str(exc_info.value) == "Status must be one of: pending, validated"
```

Comment:
A bit brittle, but the validation functions. This will be worth changing to something that doesn't break with every value added.